install_debs_into_image.sh

```diff
@@ -66,6 +66,7 @@ LOOPDEVICE=$(losetup -f)
 CONFIGTXT="$IMAGEDIR/boot/config.txt"
 
 cleanup_umount() {
+	sync
```

Comment:
Why is this needed?

Reply:
I see this sometimes:
`[2022-10-14 08:40:12]  install file:/lib/firmware/renesas_usb_fw.mem
[2022-10-14 08:40:13]  fsck.fat 4.2 (2021-01-31)
[2022-10-14 08:40:14]  There are differences between boot sector and its backup.
[2022-10-14 08:40:14]  This is mostly harmless. Differences: (offset:original/backup)
[2022-10-14 08:40:14]    65:01/00
[2022-10-14 08:40:14]    Not automatically fixing this.
[2022-10-14 08:40:14]  Dirty bit is set. Fs was not properly unmounted and some data may be corrupt.
[2022-10-14 08:40:14]   Automatically removing dirty bit.
[2022-10-14 08:40:14]  
[2022-10-14 08:40:14]  *** Filesystem was changed ***
[2022-10-14 08:40:14]  Writing changes.
[2022-10-14 08:40:14]  /dev/loop2p1: 322 files, 85960/516190 clusters
[2022-10-14 08:40:14]  /dev/loop2: [2065]:39512 (/mnt/sdb/1/2022-07-28-revpi-buster-lite.img)`

Reply:
If this change should be considered, please add a description on the circumstances this can happen.

Reply:
And if this is an issue, then we need to do this in our customize_image script as well?

Reply:
I need to object on both statements.
I don't see why this behavior might be triggered be any operation before. We do a umount, which should take care of everything.
A sync command can impact the performance of the whole system. As **all** filesystems will be synced. And thus might heavily affect the performance of a build system with multiple tasks running. Even running this in a VM can impact the performance of the host system.

Reply:
One more information, when the dirty bit occurs, the target image is located in a different mount point from where imagebakery located, that it, the imagebakery located in /dev/sda8/   and target image in /dev/sdb1
However, I tried to reproduce it with the environment above, but it doesn't happen anymore.